src/OpenLoco/Graphics/Colour.cpp

```diff
@@ -11,6 +11,41 @@ namespace OpenLoco::Colour
     loco_global<uint8_t[32][8], 0x01136BA0> _colour_map_a;
     loco_global<uint8_t[32][8], 0x01136C98> _colour_map_b;
 
+    // 0x005045FA
+    static constexpr uint8_t _translucentColourMap[31] = {
```

Comment:
If you use std::array you would get out of bounds checks for debug builds and generally better value semantics.